### ISSUE TITLE
sb989 issue#9

### ISSUE THREAD

**Issue by sb989** [ScottDiRoma/Graph#35]
https://github.com/ScottDiRoma/Graph/blob/6dfdeb39d5b6aa8a5766c29cd3c03a6230924c2f/GridGraph.java#L45
you can create an int size = grid.size() to improve readability.

**Comment by ScottDiRoma** [comment]
Yes that's very unreadable. I did something like what you suggested.
